components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -3354,6 +3429,56 @@ def listOrphanedAnnotations(self, parent_type, parent_ids, eid=None,
         for e in q.findAllByQuery(sql, p, self.SERVICE_OPTS):
             yield AnnotationWrapper._wrap(self, e)
 
+    def getAnnotationCounts(self, objDict=None):
+        """
+        Get the annotion counts for the given objects
+        """
+        obj_type = None
+        obj_ids = []
+        if objDict is not None:
+            for key in objDict:
+                for o in objDict[key]:
+                    if obj_type is not None and obj_type != key:
+                        raise AttributeError(
+                            "getAnnotationCounts cannot be used with "
+                            "different types of objects")
+                    obj_type = key
+                    obj_ids.append(o.id)
+
+        if obj_type is None:
+            return countAnnotations()
+
+        if obj_type == "project":
+            obj_type = "Project"
+        if obj_type == "dataset":
+            obj_type = "Dataset"
+        if obj_type == "image":
+            obj_type = "Image"
+        if obj_type == "screen":
+            obj_type = "Screen"
+        if obj_type == "plate":
+            obj_type = "Plate"
+        if obj_type == "plateacquisition":
+            obj_type = "PlateAcquisition"
+        if obj_type == "well":
+            obj_type = "Well"
```

Comment:
how about to replace that long if by `obj_type.title().replace("Plateacquisition", "PlateAcquisition")`